bi-connectors/PowerBIConnector/docs/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to the connector will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.1] 2022-02-28
+### Added
+- Added option to allow users to choose a certificate validation option
+### Changed
+- Changed beta to false
```

Comment:
What does changing beta to false mean for users? 

Did we leave the connector in beta erroneously?

It would be great to explain here what this step means -- dropping beta status is generally considered a big deal.

Reply:
Beta is for uncertified connectors and since we just got certified then we can now remove beta.